.github/workflows/ruby.yml

```diff
@@ -44,29 +44,37 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 10 # this might cause issues if there are more than 10 commits in a PR (?)
+
```

Comment:
(Unrelated change; I think adding newlines between the steps makes it easier to parse this list.)